libs/designsystem/src/lib/components/avatar/avatar.component.spec.ts

```diff
@@ -186,7 +186,11 @@ describe('AvatarComponent', () => {
   });
 
   describe(`when rendering Avatar with themeColor`, () => {
-    const colors = [...ColorHelper.mainColors, { name: 'white', value: '#ffffff' }];
+    const colors = [
+      ...ColorHelper.mainColors,
+      { name: 'white', value: '#ffffff' },
+      { name: 'semi-light', value: '#ebebeb' },
+    ];
```

Comment:
Consider getting from helper instead of hardcoding values:
```suggestion
    const white = getColor('white');
    const semiLight = getColor('semi-light');
    const colors = [
      ...ColorHelper.mainColors,
      white,
      semiLight,
    ];
```